interfaces/builtin/basedeclaration.go

```diff
@@ -32,15 +32,103 @@ type: base-declaration
 authority-id: canonical
 series: 16
 revision: 0
-plugs:
-  network: true
 slots:
-  network:
-    allow-installation:
-      slot-snap-type:
-        - core
+  bluetooth-control:
+    deny-auto-connection: true
```

Comment:
it will come in a PR about installation, we have no check implemented for that atm